.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -7,6 +7,7 @@ body:
       value: |
          ### ❗️ Read this before submitting your bug report:
          - **Write in English.** Reports in all other languages will be closed.
+         - **Do not open issues about following modules**  - klaviyopsautomation, mbeshipping, ps_accounts, ps_edition_basic, ps_eventbus, ps_facebook, ps_checkout, ps_mbo, ps_metrics, psxmarketingwithgoogle. Contact prestashop.com support instead.
```

Comment:
I suggest to add a generic message about addons modules

Reply:
People don't know what are addon modules. They download the bloated distro from prestashop.com thinking it's a clean installation.